### PR TITLE
Finalize Minigame F

### PR DESCRIPTION
**Pull request by irenevanderblij**
Minigame F now fully works. When the game starts painting appear in the room which on interaction show the code which the mobile players have to enter!

Minigame B is kinda broken at the moment. If you just want to test this minigame, you can edit the list of states in the class GameSession in KroketHost. If only put state F there, you can interact with the door and test this minigame. Testing the full game is also possible of course.

### CODE REVIEW

**Comment by irenevanderblij** [comment]
I refactored minigame F to be named minigame D, since this is the logical sequence after C. The Waiting minigame (before game D) is fully refactored to waiting.

Please review (and merge)

File: EscapeVR/src/main/java/nl/tudelft/kroket/escape/EscapeVR.java [hunk 1]
```diff
@@ -470,6 +474,23 @@ public void handleEvent(EventObject ev) {
           hud.setCenterText("You found login data for the computer!");
           clientThread.sendMessage("DONE[A][ADVANCE]");
           break;
+        case "D1":
+          ((EscapeScene) sceneManager.getScene("escape")).remove("D1");
```
Comment (Faffie) [inline EscapeVR/src/main/java/nl/tudelft/kroket/escape/EscapeVR.java]: Could you please get this scene once and then call its members on it?

File: EscapeVR/src/main/java/nl/tudelft/kroket/escape/EscapeVR.java [hunk 2]
```diff
@@ -470,6 +474,23 @@ public void handleEvent(EventObject ev) {
...
+          ((EscapeScene) sceneManager.getScene("escape")).addCode13("Textures/Painting/13.jpg", "D_13");
```
Comment (Faffie) [inline EscapeVR/src/main/java/nl/tudelft/kroket/escape/EscapeVR.java]: Could you please get this scene once and then call its members on it?

File: EscapeVR/src/main/java/nl/tudelft/kroket/escape/EscapeVR.java [hunk 3]
```diff
@@ -470,6 +474,23 @@ public void handleEvent(EventObject ev) {
...
+          ((EscapeScene) sceneManager.getScene("escape")).addCode37("Textures/Painting/questionmark.jpg", "D2");
```
Comment (Faffie) [inline EscapeVR/src/main/java/nl/tudelft/kroket/escape/EscapeVR.java]: Could you please get this scene once and then call its members on it?

**Comment by Faffie** [comment]
Please get the scene and cast once, also check the type before casting please.

**Comment by irenevanderblij** [comment]
I changed it so that I get the escape scene only once. I am not sure how you want me to only cast the escapescene once. Is this what you meant?

**Comment by Mayke93** [comment]
when the conflict are fixed, this can be merged

**Comment by irenevanderblij** [comment]
I solved the conflicts

**Pull request merged**